pkgs/development/python-modules/ots-python/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, substituteAll
+, opentype-sanitizer
+, setuptools-scm
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "ots-python";
```

Comment:
1 last small thing please @danc86 : Could you make both the attribute name of this package, `pname` and the file name all be `opentype-sanitizer`? This would also allow you to `inherit pname version;` in `fetchPypi`. I didn't notice this issue in my previous review, and I remember Python ecosystem maintainers are rather strict regarding this habit.

Reply:
You are right, I just noticed that now and I was about to comment. It may cause issues, and the current situation is not so bad.